src/Abp.Zero.Common/Localization/ApplicationLanguageManager.cs

```diff
@@ -52,18 +52,20 @@ public class ApplicationLanguageManager :
         /// Gets list of all languages available to given tenant (or null for host)
         /// </summary>
         /// <param name="tenantId">TenantId or null for host</param>
-        public virtual async Task<IReadOnlyList<ApplicationLanguage>> GetLanguagesAsync(int? tenantId)
+        /// <param name="includeDisabled">false: return will not include disabled languages</param>
+        public virtual async Task<IReadOnlyList<ApplicationLanguage>> GetLanguagesAsync(int? tenantId, bool includeDisabled = false)
         {
-            return (await GetLanguageDictionaryAsync(tenantId)).Values.ToImmutableList();
+            return (await GetLanguageDictionaryAsync(tenantId, includeDisabled)).Values.ToImmutableList();
         }
 
         /// <summary>
         /// Gets list of all languages available to given tenant (or null for host)
         /// </summary>
         /// <param name="tenantId">TenantId or null for host</param>
-        public virtual IReadOnlyList<ApplicationLanguage> GetLanguages(int? tenantId)
+        /// <param name="includeDisabled">false: return will not include disabled languages</param>
+        public virtual IReadOnlyList<ApplicationLanguage> GetLanguages(int? tenantId, bool includeDisabled = false)
```

Comment:
How about making it two different methods?

e.g `GetActiveLanguages` and `GetLanguages`

Reply:
Yes, it would be better. Unfortunately I didn't think of that while writing :(
I will change it

Reply:
But if I add GetActiveLanguages we have to replace `GetLanguages` with `GetActiveLanguages` in everywhere so that they dont work with disabled languages and support that. 

I am thinking to add `GetAllLanguages` which will return all languages including disabled languages. And `GetLanguages` will return only actives.

What do you think ?

Reply:
`GetAllLanguages` sounds good.

I would suggest start deprecating `GetLanguages` (and use `GetAllLanguages` as the implementation)

Then introduce `GetActiveLanguages` to be more explicit about this breaking change

Reply:
So, since I dont change `GetLanguages` logic, I can let `GetLanguages` keep return all languages and add `GetActiveLanguages` then use it anywhere I need. 
It that ok for you?

Reply:
I think the point I was trying to make was that `since we are making breaking change and it modified the default languages retrieval by active status` then we should make the method more explicit (therefore, I suggested `GetActiveLanguages`)

😀

Reply:
That being said, we can definitely go with `GetLanguages` and `GetActiveLanguages`